prog/test/hetzner_server.rb

```diff
@@ -85,18 +85,32 @@ def self.assemble
   label def wait_setup_host
     reap
 
-    hop_test_host if children_idle
+    hop_test_host_encrypted if children_idle
 
     donate
   end
 
-  label def test_host
-    bud Prog::Test::VmGroup
+  label def test_host_encrypted
+    strand.add_child(Prog::Test::VmGroup.assemble(storage_encrypted: true))
```

Comment:
Now that we have parameters for `assemble`, changing to explicitly calling `.assemble` instead of using `bud` so we don't make assumptions about how `VmGroup`'s frame is structured.